packages/test/test-end-to-end-tests/src/test/blobs.spec.ts

```diff
@@ -241,6 +242,20 @@ describeNoCompat("blobs", (getTestObjectProvider) => {
 
 		const attachOpP = new Promise<void>((resolve, reject) =>
 			container1.on("op", (op) => {
+				if (op.contents?.type === "groupedBatch") {
+					const messages = op.contents.contents as IBatchMessage[];
+					for (const message of messages) {
+						if (
+							message.contents &&
+							JSON.parse(message.contents).type === ContainerMessageType.BlobAttach
+						) {
+							// eslint-disable-next-line @typescript-eslint/no-unused-expressions
+							message.metadata?.blobId
```

Comment:
The whole purpose of metadata.blobId is to ensure that storage can see this reference. If we do grouping, storage loses ability to observe that metadata.
Basically, blob attach ops should never be grouped. And probably grouping code should assert that ops that it groups have no metadata.

Reply:
This would cause a change to the `BatchManager` because it adds the batch metadata upon popping a batch. Are there likely to be other op types that storage needs to see the metadata of?